ctat_mutations

```diff
@@ -4,635 +4,332 @@
+STR_STAR_PROGRESS = "Log.progress.out"
+
+# Variant Calling modes - GATK or NONE
+STR_VARIANT_GATK = "GATK"
+STR_VARIANT_NONE = "NONE"
+LSTR_VARIANT_CALLING_CHOICES = [STR_VARIANT_GATK,
+                                STR_VARIANT_NONE]
+
+# PICARD platforms
 # Choices for platform
 STR_ILLUMINA = "ILLUMINA"
 LSTR_SEQ_CHOICES = [STR_ILLUMINA, "SLX,SOLEXA",
                     "SOLID,454", "COMPLETE",
                     "PACBIO", "IONTORRENT",
                     "CAPILLARY", "ONT"]
 
-# Choices for alignment
-#STR_ALIGN_GSNP = "GSNAP"
-STR_ALIGN_STAR = "STAR"
-STR_ALIGN_STAR_LIMITED = "LIMITED"
-LSTR_ALIGN_CHOICES = [STR_ALIGN_STAR, STR_ALIGN_STAR_LIMITED]
-
-# Choices for variant calling
-STR_VARIANT_GATK = "GATK"
-STR_VARIANT_SAMTOOLS = "SAM"
-STR_VARIANT_NONE = "NONE"
-LSTR_VARIANT_CALLING_CHOICES = [STR_VARIANT_GATK,
-                                STR_VARIANT_NONE]
-# variant calling methods 
+# GATK variant callers
 STR_GATK_HC = "HaplotypeCaller"
-STR_GATK_M2 = "Mutect2"
+STR_GATK_MUTECT2 = "Mutect2"
 LSTR_VARIANT_METHOD_CHOICES = [STR_GATK_HC,
-                                STR_GATK_M2]
+                                STR_GATK_MUTECT2]
 
 # Choices for variant filtering
 STR_FILTERING_BCFTOOLS = "BCFTOOLS"
-STR_FILTERING_GATK = "GATK"
+STR_FILTERING_HC_REGULAR = "HC_REGULAR"
+STR_FILTERING_HC_CNN = "HC_CNN"
+STR_FILTERING_MUTECT2_REGULAR = "MUTECT2_REGULAR"
+STR_FILTERING_MUTECT2_LROM = "MUTECT2_LROM"
 STR_FILTERING_NONE = "NONE"
 
-LSTR_VARIANT_FILTERING_CHOICES = [STR_FILTERING_GATK,
+LSTR_VARIANT_FILTERING_CHOICES = [STR_FILTERING_HC_REGULAR,
+                                  STR_FILTERING_HC_CNN,
+                                  STR_FILTERING_MUTECT2_REGULAR,
+                                  STR_FILTERING_MUTECT2_LROM,
+                                  STR_FILTERING_BCFTOOLS,
                                   STR_FILTERING_NONE]
 
-# This mode is used in validating the method in the context fo DNA-seq
-# It is not intended to be ran on biological samples for studies.
-STR_DNASEQ_VALIDATION = "DNASEQ"
+STR_FILTERING_GATK = [STR_FILTERING_HC_REGULAR,
+                                  STR_FILTERING_HC_CNN,
+                                  STR_FILTERING_MUTECT2_REGULAR,
+                                  STR_FILTERING_MUTECT2_LROM
+                   ]
 
-# Named files for pipeline
-C_STR_CANCER_TAB = "cancer.tab"
+# File names required for filtering
+C_STR_INIT_FILTER = "variants_initial_filtering.vcf"
+
+# File name for cosmic filtering
 C_STR_CANCER_ANNOTATED_VCF = "variants_annotated.vcf.gz"
+
+# File name for CRAVAT filtering
 C_STR_CANCER_VCF = "cancer.vcf"
-C_STR_INIT_FILTER = "variants_initial_filtering.vcf"
+C_STR_CANCER_TAB = "cancer.tab"
 
-# CRAVAT related
+# Files for CRAVAT
+STR_CRAVAT_CLASSIFIER_DEFAULT = "Other"
 I_CRAVAT_ATTEMPTS = 180
 I_CRAVAT_WAIT = 60
-STR_CRAVAT_CLASSIFIER_DEFAULT = "Other"
 STR_FDR_CUTTOFF = "0.3"
-STR_CRAVAT_HEADER_INFO = os.path.join(BASEDIR, "meta/cravat_annotation.txt")
```

Comment:
this cravat_annotation.txt file used to be included in the mutation lib, but I moved it to github here.  Are you removing it again?  Might need to restore this.  It made more sense to keep it in the repo than in the mutation lib given that it worked for both the hg19 and hg38 w/o modification, and it was small enough.